packages/contracts/src/utils/core_combinatorial_utils.ts

```diff
@@ -145,13 +145,39 @@ export class CoreCombinatorialUtils {
     public exchangeWrapper: ExchangeWrapper;
     public assetWrapper: AssetWrapper;
     public static generateFillOrderCombinations(): FillScenario[] {
-        const takerScenarios = [TakerScenario.Unspecified];
-        const feeRecipientScenarios = [FeeRecipientAddressScenario.EthUserAddress];
-        const makerAssetAmountScenario = [OrderAssetAmountScenario.Large];
-        const takerAssetAmountScenario = [OrderAssetAmountScenario.Large];
-        const makerFeeScenario = [OrderAssetAmountScenario.Large];
-        const takerFeeScenario = [OrderAssetAmountScenario.Large];
-        const expirationTimeSecondsScenario = [ExpirationTimeSecondsScenario.InFuture];
+        const takerScenarios = [
```

Comment:
Maybe we should introduce a config variable instead of commenting out the code?

Reply:
The point of the commented out code here is to elucidate the options that are not being tested, and to make it very easy to test them locally at will. I do see your point that this could become out-of-date, but it is still greppable, so I think we should take the risk.